spacy_llm/tasks/raw/registry.py

```diff
@@ -0,0 +1,56 @@
+from typing import Optional, Type
+
+from ...registry import registry
+from ...ty import ExamplesConfigType, FewshotExample, ShardMapper, ShardReducer
+from ...ty import TaskResponseParser
+from ..util.sharding import make_shard_mapper
+from .parser import parse_responses_v1
+from .task import DEFAULT_RAW_TEMPLATE_V1, RawTask
+from .util import RawExample, reduce_shards_to_doc
+
+
+@registry.llm_misc("spacy.RawParser.v1")
+def make_lemma_parser() -> TaskResponseParser[RawTask]:
+    return parse_responses_v1
+
+
+@registry.llm_misc("spacy.RawShardReducer.v1")
+def make_shard_reducer() -> ShardReducer:
+    return reduce_shards_to_doc
+
+
+@registry.llm_tasks("spacy.Raw.v1")
+def make_raw_task(
+    template: str = DEFAULT_RAW_TEMPLATE_V1,
+    field: str = "reply",
```

Comment:
Considering that this is a field on the `Doc` level, maybe something slightly more specific, like `llm_response` or `llm_reply` or such?